deps/build.jl

```diff
@@ -6,4 +6,4 @@ using Conda
 
 pip = joinpath(Conda.BINDIR, "pip")
 run(`$pip install pyyaml setuptools-scm`)
-run(`$pip install git+https://github.com/ScottishCovidResponse/data_pipeline_api@0.5.0`)
+run(`$pip install git+https://github.com/ScottishCovidResponse/data_pipeline_api@0.6.0`)
```

Comment:
The PRoject.toml version could be bumped to 0.6.0 to match data_pipeline_api (it's 0.4.0 atm), not sure what best practice is when doing a wrapper, probably it doesn't make sense to match  in general. 

However, think you do need to bump the version otherwise Simulation.jl won't see the changes

Reply:
Yep good catch, bumped to 0.4.1